<launch>
    <group ns="$(env HOSTNAME)" >
        <node name="fc" pkg="ouijabot" type="force_control.py" output="screen">
		<param name="maxDelay" value=".5"/>
	        <param name="maxVel" value="3"/>
		<param name="readFrq" value="10"/>
		<param name="cmdFrq" value="100"/>
		<param name="currFrq" value="20" />
	</node>
    </group>
</launch>
